Package.swift

```diff
@@ -26,6 +26,7 @@ let package = Package(
                         // Windows does not use POSIX mutex
                         .when(platforms: [.linux, .macOS])),
                 .define("MDB_USE_ROBUST", to: "0"),
+                .unsafeFlags(["-Wno-shorten-64-to-32"])
```

Comment:
If I understand [the `unsafeFlags(_:_:)` documentation](https://developer.apple.com/documentation/packagedescription/swiftsetting/unsafeflags(_:_:)) correctly, this change would prevent other packages from using the CLMDB library product:

> As some build flags can be exploited for unsupported or malicious behavior, the use of unsafe flags makes the products containing this target ineligible for use by other packages.